web/src/stories/group-item/base.stories.tsx

```diff
@@ -56,8 +56,9 @@ export const Type = (): JSX.Element => (
 )
 
 export const Truncated = (): JSX.Element => (
-  <Flex gap="2x" direction="column" style={{ width: 200 }} wrap>
+  <Flex gap="2x" direction="column" style={{ width: 500 }} width="100%" wrap>
```

Comment:
this `style={{ width: 500 }} width="100%"` looks ugly, what do you think?

Reply:
Let's remove `maxWidth` property from AvatarItem and GroupItem, then we will control their length via parent width property. That's why I think we need to keep this, or just create a parent div element and apply width there, to show that element will be limited to parent width

Reply:
I added here width: 200, to show that truncating also works for name